iron-selectable.html

```diff
@@ -336,10 +346,19 @@
       if (this.selectedAttribute) {
         this.toggleAttribute(this.selectedAttribute, isSelected, item);
       }
+      if (this.selectedPropertyValue) {
+        item[this.selectedAttribute] = this.selectedPropertyValue;
+      }
       this._selectionChange();
       this.fire('iron-' + (isSelected ? 'select' : 'deselect'), {item: item});
     },
 
+    _updatePropertyValue: function(selectedItem, selectedAttribute, propertyValue) {
+      if (selectedAttribute && propertyValue) {
+        selectedItem[selectedAttribute] = propertyValue;
```

Comment:
This crashes on first load as `selectedItem` is undefined.
Works with `if (selectedItem && ...)` on the line above.